.github/dependabot.yml

```
@@ -15,9 +18,12 @@ updates:
   - package-ecosystem: github-actions
     directory: /
     schedule:
-      interval: weekly
-      time: '00:00'
+      interval: monthly
     open-pull-requests-limit: 10
+    labels:
+      - ci
+      - dependency
+      - github_actions
```

```suggestion
      - github-actions
```
due to jellyfin/jellyfin-plugin-meta#9